README.md

```diff
@@ -1 +1,5 @@
-# Webpack Template
+# Webpack Starter Template
+
+<img src="./public/logo.svg" width="100" height="100" alt="WarpJS" style="background-color:#000;border-radius:50%;padding:15px;">
+
+Template for building a basic [webpack](https://webpack.js.org/) project with [WarpJS](https://warpjs.com/)
```

Comment:
In fact, the template is to build a basic project with webpack. There is no warp inside, right?

Reply:
exactly